src/main/java/net/mcreator/blockly/data/StatementInput.java

```diff
@@ -24,7 +24,7 @@
 
 	public boolean disable_local_variables;
 
-	@Override public String getName() {
+	@Override public String name() {

```

Comment:
Is it not possible to make new record constructor here including disable_local_variables? Did not test if this is possible in Java yet, just thinking aloud

Reply:
Since all record fields are final by default, I should instead make a constructor that assigns this one to its default value. However, if this field is modified, the answer is **Just no**.

Reply:
I don't think this field is modified

Reply:
But inputs are handled by GSON afaik

Reply:
To turn statements/advanced inputs into records, I'd need them to not extend any other classes, which would require another PR refactoring them.
And I got no conflicts with `IInput`'s JSON adapter, it works both before and now.

Reply:
No need to change this then